src/server.c

```diff
@@ -5587,10 +5587,10 @@ sds genRedisInfoString(dict *section_dict, int all_sections, int everything) {
             "redis_version:%s\r\n", REDIS_VERSION,
             "server_name:%s\r\n", SERVER_NAME,
             "server_version:%s\r\n", SERVER_VERSION,
-            "redis_git_sha1:%s\r\n", serverGitSHA1(),
```

Comment:
Afaik, we are not removing any Redis strings from info commands just yet. Only adding a new server string.